klasses/api_test.py

```diff
@@ -177,3 +185,249 @@ def test_fetch_bootcamp_run_dates(end_date_params):
     assert fetch_bootcamp_run(run.display_title) == run
     with pytest.raises(BootcampRun.DoesNotExist):
         fetch_bootcamp_run("invalid")
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize("novoed_integration", [True, False])
+def test_create_run_enrollments(mocker, user, settings, novoed_integration):
```

Comment:
`create_run_enrollment` is now the function that has the critical functionality, and `create_run_enrollments` is just a helper that calls that function and aggregates results. We should change this test case to test `create_run_enrollment` directly, then add a test case for `create_run_enrollments` which patches `create_run_enrollment`